src/_includes/layouts/components/head.njk

```diff
@@ -48,23 +52,25 @@
 
 <meta name="robots" content="noai, noimageai, {{ metaRobots }}">
 <meta name="author" content="{{ metaAuthor }}">
-<meta property="og:site_name" content="{{ metaAuthor }}">
-<meta name="twitter:title" property="og:title" content="{{ metaAuthor }}">
-<meta name="twitter:image:alt" property="og:image:alt" content="{{ metaAuthor }}">
-<meta name="description" content="{{ metaDescription }}"/>
-<meta name="twitter:description" property="og:description" content="{{ metaDescription }}">
-<meta name="twitter:site" content="@{{ metaAuthor }}">
-<meta name="twitter:creator" content="@{{ metaAuthor }}">
-<meta name="twitter:image" property="og:image" content="{{ metaImg }}">
-<meta name="twitter:url" property="og:url" content="{{ metaUrl }}">
+<meta name="generator" content="{{ eleventy.generator }}">
+<meta http-equiv="x-ua-compatible" content="ie=edge">
+
 <link rel="canonical" href="{{ metaUrl }}">
+<meta property="og:url" content="{{ metaUrl }}">
+
+<meta property="og:site_name" content="{{ metaAuthor }}">
+<meta property="og:title" content="{{ metaTitle }}">
+
+<meta name="description" content="{{ metaDescription }}">
+<meta property="og:description" content="{{ metaDescription }}">
+<meta property="og:image:alt" content="{{ metaTitle.split(' |')[0] | trim | replace('\\.', '') }}.">
```

Comment:
I think I'm a little confused about what this is doing

Reply:
For the alt text description, use `meta.title`, removing the pipe (`|`) along with the space before it and any text after. Then, remove any trailing period from the title and ensure the final alt text always ends with a single period. 

For example, the home page title `"Free online courses on design and development."` ends with a period. Without cleaning it, the alt text would end with a double period (`..`), so the final output should be: `"Free online courses on design and development."`